src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__photo">
+        <img
+          class="photo"
+          src="./images/imac.jpeg"
+          alt="Monoblock APPLE A1419 iMac">
+      </div>
+
+      <h2 class="product-card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K   (MNED2UA/A)
+      </h2>
+
+      <div class="product-card__code">
+        Код товара: 195434
+      </div>
+
+      <div class="product-card__rate">
+        <div class="stars-4">
```

Comment:
you cant use modifiers without block name, also modifiers should use `--` as a splitter between name and value, e.g. `stars--4`
```suggestion
        <div class="stars stars--4">
```